src/software/SfM/main_openMVG2CMPMVS.cpp

```diff
@@ -77,7 +77,11 @@ bool exportToCMPMVSFormat(
 
       const Pose3 pose = sfm_data.GetPoseOrDie(view);
       Intrinsics::const_iterator iterIntrinsic = sfm_data.GetIntrinsics().find(view->id_intrinsic);
-
+      const IntrinsicBase * cam = iterIntrinsic->second.get();
+      if(cam->w() == 0 || cam->h() == 0)
```

Comment:
This is strange to have image with size 0.
Perhaps better to check if iterIntrinsic is defined or not